qdrant-landing/content/blog/case-study-dragonfruit.md

```diff
@@ -0,0 +1,83 @@
+
+* Operate a vector store at enterprise scale: thousands of locations → thousands of cameras, accumulating into tens to hundreds of billions of vectors and multi-terabyte storage.
+### Why Qdrant: Performance headroom and operational control
+
+Dragonfruit chose the open-source version of Qdrant as its vector database to meet the twin pressures of real-time reads and high-velocity writes. In head-to-head experiments, Qdrant delivered the QPS targets they needed while giving the team granular, [per-collection](https://qdrant.tech/documentation/concepts/collections/) tuning to match workload diversity.
+
+Key reasons the team highlighted:
+
+* **Per-collection configurability.** Collections with heavy reads and low writes use different settings than write-heavy pipelines. Tuning shard counts and HNSW parameters by collection helped hit latency Service Level Objectives without overprovisioning.
+
+* **Efficient numeric formats.** For most vision workloads, [float16](https://qdrant.tech/documentation/concepts/vectors/) vectors were sufficient, improving memory efficiency and cache behavior with no material loss in retrieval accuracy for their use cases.
+
+* **Open source and ecosystem fit.** Qdrant’s OSS model aligned with Dragonfruit’s platform strategy and let them co-evolve the deployment with their edge and cloud stack.
+
+*“With Qdrant’s collection-level controls, we matched very different workloads, some ingestion-heavy, some read-intensive, and still hit real-time query performance.”*  
+ — **Shivang Agarwal, VP Engineering, Dragonfruit AI**
+### Solution architecture: Split AI \+ vector search
+
+At the edge, Mac Minis ingest RTSP streams from IP cameras, run on-prem inference, and emit compact embeddings and event metadata. In the cloud, Qdrant serves multiple, distinct retrieval patterns:
+
+* **Multi-camera person re-ID.** Embeddings link tracks across cameras to build “spaghetti charts” of in-store movement, enabling spatial analytics and alerting.
+
+* **Self-checkout product verification.** Frames around scan events are embedded and matched against clustered product libraries to detect missed or incorrect scans in real time.
+
+* **Full-frame semantic search.** Video frames are embedded for [text-to-image retrieval](https://qdrant.tech/advanced-search/) (for example “emergency door open”), enabling rapid incident triage and safety reviews.
+### Results: New agents, faster delivery, lower TCO
+
+Qdrant became an enabling layer for Dragonfruit’s agent roadmap. With real-time retrieval performance and cost-efficient storage, the team launched and iterated on new domain-specific agents more quickly, spanning loss prevention, occupational safety, and warehouse operations.
+
+From a go-to-market perspective, Dragonfruit focuses on enterprise buyers and also works through major IT and security integrators, giving them reach into retail (their largest segment), manufacturing, government, entertainment, and healthcare. As Price summarized:
```

Comment:
How does Dragonfruit bring new customers to the platform? Do different customers share the same infrastructure? If so, it would make sense to describe how they approach multitenancy.